samples/Samples.AspNetCoreMvc21/OtelSdkPlugin.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using OpenTelemetry.Trace;
+using Samples.AspNetCoreMvc.Controllers;
+
+namespace Samples.AspNetCoreMvc
+{
+    public class OtelSdkPlugin
+    {
+        public TracerProviderBuilder ConfigureTracerProvider(TracerProviderBuilder builder)
```

Comment:
When we are more confident in this interface we should go back to SDK and add it to the instrumentations and ask people implementing instrumentations to add it.

Reply:
I will add it to the design docs